content/3.workflow/4.payment-initiation.md

```diff
@@ -1,8 +1,11 @@
 # 3. Payment initiation
 
-1. The Participant initiates the payment transaction off-chain via a Web UI. The payment request is handled by a web server on the Participant’s node of the Corda business network and sent via Corda RPC to the custom Payment CorDapp installed on the Participant’s node on-chain.
+1. The Participant initiates the payment transaction off-chain via a Web UI. 
+The payment request is handled by a web server on the Participant’s node of the Corda business network and sent via Corda RPC to the custom Payment CorDapp installed on the Participant’s node on-chain.
+Every singular payment gets it's own unique UUID, which can be used to find it, if necessary.
```

Comment:
Let's unite back these lines into single one, so it will be easily to identify new content.

```suggestion
1. The Participant initiates the payment transaction off-chain via a Web UI. The payment request is handled by a web server on the Participant’s node of the Corda business network and sent via Corda RPC to the custom Payment CorDapp installed on the Participant’s node on-chain. Every singular payment gets it's own unique UUID, which can be used to find it, if necessary.
```